src/sentry/workflow_engine/handlers/condition/slow_condition_query_handlers.py

```diff
@@ -460,6 +474,7 @@ def batch_query(
             model = get_issue_tsdb_group_model(category)
             # InvalidFilter should not be raised for errors
             results = self.get_chunked_result(
+                tsdb_function=tsdb.backend.get_sums,
```

Comment:
Nice, this seems simpler and you don't really need to declare them on the class level anyway.